package.json

```diff
@@ -267,5 +271,11 @@
       "commit-msg": "commitlint --verbose -e $HUSKY_GIT_PARAMS",
       "pre-commit": "npm run --silent srcdist && npm run --silent lint:staged"
     }
+  },
+  "standard-version": {
+    "scripts": {
```

Comment:
What happened to the `prerelease` step?

Reply:
`npm run build` is where `webex.version` is set and i refactored the babel-plugin to use the package.json instead of pkg-up

Reply:
OK, I'm ok with that. Let's steps, less bugs! 😁